sdks/java/io/hbase/src/main/java/org/apache/beam/sdk/io/hbase/HBaseIO.java

```diff
@@ -679,6 +667,5 @@ public void populateDisplayData(DisplayData.Builder builder) {
         }
     }
 
-    public static final Coder<KV<byte[], Iterable<Mutation>>> WRITE_CODER =
-            KvCoder.of(ByteArrayCoder.of(), IterableCoder.of(HBaseMutationCoder.of()));
+    public static final Coder<Mutation> WRITE_CODER = HBaseMutationCoder.of();
```

Comment:
While we're at simplifying things.. I believe you can remove this and all the setCoder calls by using something like https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryCoderProviderRegistrar.java